lib/srv/db/cloud/watchers/rds.go

```diff
@@ -106,6 +106,14 @@ func (f *rdsDBInstancesFetcher) getRDSDatabases(ctx context.Context) (types.Data
 	}
 	databases := make(types.Databases, 0, len(instances))
 	for _, instance := range instances {
+		if !services.IsRDSInstanceSupported(instance) {
```

Comment:
What about Aurora clusters? Does Aurora provide MariaDB engine?

Reply:
I don't see MariaDB in Aurora (checked website + docs).